23.2.22/classone3.js

```diff
@@ -0,0 +1,21 @@
+class Classengg {
+    constructor(branch, semester, subject, year, result){
+        this.branch = branch ;
+        this.semester = semester;
+        this.subject = subject;
+        this.year = year ;
+        this.result = result ;
+
+    }
+    getBranch () {
+        return `${this.branch} is from semester ${this.semester}`;
+    }
+        
+}
+const myClassengg = new Classengg ("mechanical","third", "som", "2014","pass");
+
+console.log(myClassengg.branch);
+console.log(myClassengg.semester);
+console.log (myClassengg.subject);
```

Comment:
getBranch () {
        return `${this.name} ${this.branch} is from semester ${this.semester} persuing subject ${this.subject}  passesd in the year ${this.year} and the grade is ${this.result}`;
    }

}
const myClassengg = new Classengg ("saraubh","mechanical branch","third", "som", "2014","pass");

console.log(myClassengg.getBranch());
        

define properly in the method and call function @ramprakash12345